### PR TITLE
Update example.html

### PR DESCRIPTION
**Pull request by NARESHSDE**

### CODE REVIEW

File: example.html [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <!DOCTYPE html>
-<html lang="en">
```
Comment (maverick-m8R) [inline example.html:2]: Too many Repetitions and open tags.

File: example.html [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+<html lang='en'>
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="author" content="CJ Patoilo" />
     <meta
       name="description"
-      content="Milligram provides a minimal setup of styles for a fast and clean starting point. Specially designed for better performance and higher productivity with fewer properties to reset resulting in cleaner code."
+      content="Milligram provides a minimal setup of styles for a fast and clean starting point.  Specially designed for better performance and higher productivity with fewer properties to reset resulting in cleaner code."
```
Comment (cjpatoilo) [inline example.html:9]: Why use 2 whitespaces?

File: example.html [hunk 3]
```diff
@@ -1,12 +1,12 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang='en'>
```
Comment (cjpatoilo) [inline example.html:2]: Keep the pattern. Please, don't use a single quote for the attribute value.

File: example.html [hunk 4]
```diff
@@ -267,8 +264,7 @@ <h5 class="title">Examples</h5>
               title="Browser Support"
               >Browser Support</a
             >
-          </li>
-        </ul>
+          </li></ul>
```
Comment (cjpatoilo) [inline example.html:267]: Why you don't use a break line?

**Review by cjpatoilo — COMMENTED** [review]
I don't understand what motivates making these changes. 
For now, I will close this PR, but feel free to reopen or open a new one.

**Pull request closed without merging**